react-stopwatch-component/src/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class Stopwatch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isRunning: false,
+      seconds: 0,
+      icon: 'fas fa-play'
```

Comment:
since `isRunning` is already stored in state, you don't need to store the icon class. It can be computed in `render()`

Reply:
Please remove this from state if it's not needed anymore